modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -223,26 +219,33 @@ class ConnectionAvgStats {
             return;
         }
 
+        if (!data.transport || !data.transport.length) {
```

Comment:
Right, I was just wondering why that's the case. Probably for the non-bundle case, which is no longer used.